### PR TITLE
task: RBGO-124 Document business processes for admin

### PR DESCRIPTION
**Pull request by justinhamadeunity**
Updated wallet funding with NETs prerequisites https://multilot-products.github.io/fotolotto-snapchance-api-docs/#wallet-funding

Update Admin steps for creating lotteries
https://multilot-products.github.io/fotolotto-snapchance-api-docs/#creating-lotteries-and-drawings

### CODE REVIEW

File: source/includes/_flows.md.erb [hunk 1]
```diff
@@ -73,14 +85,27 @@ If the query param `responseCode` is `OK`, we can verify the deposit by making a
...
+### Process
+
+<%= image_tag "images/snapchance-admin.png" %>
+
 <aside class="warning">
 SnapChance currently only supports running a single lottery and a single active drawing at a time.  Please do not create mutliple lotteries with active drawings
 </aside>
 
-### Process
+### 1. Add your License
```
Comment (dkaga) [inline source/includes/_flows.md.erb]: Is there only one step?

**Review by dkaga — APPROVED** [review]

File: source/includes/_flows.md.erb [hunk 2]
```diff
@@ -33,27 +33,39 @@ The query param `code` should be passed in a POST request body to the API endpoi
...
 
 ## Wallet Funding
 
 A player add funds flow could look something like the following flow diagram:
 
 <%= image_tag "images/snapchance-add-funds.jpg" %>
 
-### Pre-requistes
-### Redirect URL
+### Pre-requisites
```
Comment (ron-bierman-unity) [inline source/includes/_flows.md.erb]: I think this should be: `##`

File: source/includes/_flows.md.erb [hunk 3]
```diff
@@ -73,14 +85,27 @@ If the query param `responseCode` is `OK`, we can verify the deposit by making a
 
 Using the SnapChance admin panel you can create add your gaming license as well as create lotteries and drawings using your license.
 
+### Process
```
Comment (ron-bierman-unity) [inline source/includes/_flows.md.erb]: I think this should be: `##`

**Review by ron-bierman-unity — APPROVED** [review]
Please check the headers and remove the `source/index.html.md.erb` file

Except from that LGTM

**Pull request merged**